<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>761</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>645</width>
    <height>761</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>645</width>
    <height>761</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="game_tab">
       <attribute name="title">
        <string>Play game</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,2,2,10" columnstretch="2,1,1,1,1">
        <item row="0" column="0" rowspan="3">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Player moves</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3" rowstretch="1,1" columnstretch="2,1">
           <item row="0" column="0">
            <widget class="QRadioButton" name="player_1_button">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Player 1</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <property name="autoExclusive">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QRadioButton" name="player_2_button">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Player 2</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1" rowspan="2">
            <widget class="QLabel" name="player_stone_image">
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap>img/black.png</pixmap>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="3" rowspan="2">
         <widget class="QLCDNumber" name="counter"/>
        </item>
        <item row="1" column="4">
         <widget class="QPushButton" name="reset_button">
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="time_limit_label_2">
          <property name="text">
           <string>Time limit</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="player_1_name_label_2">
          <property name="text">
           <string>Player 1</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="player_2_name_label_2">
          <property name="text">
           <string>Player 2</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QPushButton" name="clear_button">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" rowspan="2">
         <widget class="QLCDNumber" name="player_1_score"/>
        </item>
        <item row="1" column="2" rowspan="2">
         <widget class="QLCDNumber" name="player_2_score"/>
        </item>
        <item row="3" column="0" colspan="5">
         <widget class="MatplotlibWidget" name="grid" native="true"/>
        </item>
        <item row="0" column="4">
         <widget class="QPushButton" name="start_button">
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="settings_tab">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0" rowspan="3" colspan="5">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Player settings</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="4" column="1">
            <widget class="QTextEdit" name="player_2_url_textEdit"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="player_1_name_label">
             <property name="text">
              <string>Player 1 Name</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="player_2_name_label">
             <property name="text">
              <string>Player 2 Name</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="player_2_name_lineEdit">
             <property name="text">
              <string>Player 2</string>
             </property>
             <property name="maxLength">
              <number>8</number>
             </property>
             <property name="cursorPosition">
              <number>8</number>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="player_1_name_lineEdit">
             <property name="text">
              <string>Player 1</string>
             </property>
             <property name="maxLength">
              <number>8</number>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QTextEdit" name="player_1_url_textEdit"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="player_2_url_label">
             <property name="text">
              <string>Player 2 url</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="player_1_url_label">
             <property name="text">
              <string>Player 1 url</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="player_1_api_label">
             <property name="text">
              <string>Player 1 API Key</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QTextEdit" name="player_1_api_textEdit"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="player_2_api_label">
             <property name="text">
              <string>Player 2 API Key</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QTextEdit" name="player_2_api_textEdit"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="0" colspan="5">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Game settings</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QLabel" name="time_limit_label">
             <property name="text">
              <string>Time limit</string>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Seconds</string>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QSpinBox" name="spinBox_3">
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>60</number>
             </property>
             <property name="value">
              <number>15</number>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Minutes</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QSpinBox" name="spinBox_2"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="number_of_games_label">
             <property name="text">
              <string>Number of games</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2" colspan="2">
            <widget class="QSpinBox" name="spinBox">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>99</number>
             </property>
             <property name="value">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Size</string>
             </property>
            </widget>
           </item>
           <item row="1" column="5">
            <widget class="QSpinBox" name="spinBox_4">
             <property name="value">
              <number>17</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MatplotlibWidget</class>
   <extends>QWidget</extends>
   <header>matplotlibwidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
